### ISSUE TITLE
Null Aware Operation Errors

### ISSUE THREAD

**Issue by bbrosius** [tneotia/html-editor-enhanced#54]
**Describe the bug**
I'm getting null-aware operation warnings when running my app. 

**To Reproduce**
Add build 2.1.1 of html-editor-enhanced, run app. 

The following shows in the logs. 

```
./../Development/flutter/.pub-cache/hosted/pub.dartlang.org/html_editor_enhanced-2.1.1/lib/src/widgets/toolbar_widget.dart:1830:74: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
                                                    result!.files.single.name!;
                                                                         ^
../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/html_editor_enhanced-2.1.1/lib/src/widgets/toolbar_widget.dart:1982:74: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
                                                    result!.files.single.name!;
                                                                         ^

../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/html_editor_enhanced-2.1.1/lib/src/widgets/toolbar_widget.dart:2134:74: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
                                                    result!.files.single.name!;
                                                                         ^
../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/html_editor_enhanced-2.1.1/lib/src/widgets/toolbar_widget.dart:2286:74: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
                                                    result!.files.single.name!;
                                                                         ^
```

**Comment by tneotia** [comment]
I saw this yesterday, it is because of the new version of the `file_picker` plugin. I'll push a fix today, for now it shouldn't cause any problems since it is only a warning. Thanks for the report.

**Comment by tneotia** [comment]
Fixed via 8923d247aced2bf9ca3c0fde39182b95fad6f781.
